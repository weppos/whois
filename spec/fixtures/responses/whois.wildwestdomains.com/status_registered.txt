Domain Name: WILDWESTDOMAINS.COM
Registrar URL: http://www.wildwestdomains.com
Updated Date: 2011-11-01 16:31:47
Creation Date: 2000-08-22 18:29:11
Registrar Expiration Date: 2021-11-01 06:59:59
Registrar: Wild West Domains, LLC
Reseller: WWDomains.com
Registrant Name: Wild West Domains Wild West Domains
Registrant Organization: Wild West Domains
Registrant Street: 14455 N Hayden Rd Suite 219
Registrant City: Scottsdale
Registrant State/Province: Arizona
Registrant Postal Code: 85260
Registrant Country: United States
Admin Name: Wild West Domains Wild West Domains
Admin Organization: Wild West Domains
Admin Street: 14455 N Hayden Rd Suite 219
Admin City: Scottsdale
Admin State/Province: Arizona
Admin Postal Code: 85260
Admin Country: United States
Admin Phone: +1.4805058800
Admin Fax: +1.4805058844
Admin Email: dns@wildwestdomains.com
Tech Name: Wild West Domains Wild West Domains
Tech Organization: Wild West Domains
Tech Street: 14455 N Hayden Rd Suite 219
Tech City: Scottsdale
Tech State/Province: Arizona
Tech Postal Code: 85260
Tech Country: United States
Tech Phone: +1.4805058800
Tech Fax: +1.4805058844
Tech Email: dns@wildwestdomains.com
Name Server: CNS1.SECURESERVER.NET
Name Server: CNS2.SECURESERVER.NET
Name Server: CNS3.SECURESERVER.NET

****************************************************
See Business Registration Listing
****************************************************
Copy and paste the link below to view additional details:
http://who.securepaynet.net/whoischeck.aspx?domain=WILDWESTDOMAINS.COM&prog_id=26domains

The data contained in this Registrar's Whois database, 
while believed by the registrar to be reliable, is provided "as is"
with no guarantee or warranties regarding its accuracy. This information 
is provided for the sole purpose of assisting you in obtaining 
information about domain name registration records. Any use of
this data for any other purpose is expressly forbidden without
the prior written permission of this registrar.  By submitting an
inquiry, you agree to these terms of usage and limitations of warranty.
In particular, you agree not to use this data to allow, enable, or
otherwise make possible, dissemination or collection of this data, in 
part or in its entirety, for any purpose, such as the transmission of 
unsolicited advertising and solicitations of any kind, including spam. 
You further agree not to use this data to enable high volume, automated 
or robotic electronic processes designed to collect or compile this data 
for any purpose, including mining this data for your own personal or 
commercial purposes.

Please note: the owner of the domain name is specified in the "registrant" section. 
In most cases, the Registrar is not the owner of domain names listed in this database.

