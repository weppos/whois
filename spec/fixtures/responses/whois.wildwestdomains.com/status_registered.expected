#domain
  %s == "wildwestdomains.com"

#domain_id
  %s == nil


#status
  %s == :registered

#available?
  %s be_falsey

#registered?
  %s be_truthy


#created_on
  %s %CLASS{time}
  %s %TIME{2000-08-22 18:29:11}

#updated_on
  %s %CLASS{time}
  %s %TIME{2011-11-01 16:31:47}

#expires_on
  %s %CLASS{time}
  %s %TIME{2021-11-01 06:59:59}


#registrar
  %s %CLASS{registrar}
  %s.id           == nil
  %s.name         == "Wild West Domains, LLC"
  %s.organization == "Wild West Domains, LLC"
  %s.url          == "http://www.wildwestdomains.com"


#registrant_contacts
  %s %CLASS{array}
  %s.size eq(1)
  %s[0] %CLASS{contact}
  %s[0].type          == Whois::Record::Contact::TYPE_REGISTRANT
  %s[0].id            == nil
  %s[0].name          == "Wild West Domains Wild West Domains"
  %s[0].organization  == "Wild West Domains"
  %s[0].address       == "14455 N Hayden Rd Suite 219"
  %s[0].city          == "Scottsdale"
  %s[0].zip           == "85260"
  %s[0].state         == "Arizona"
  %s[0].country       == nil
  %s[0].country_code  == "United States"
  %s[0].phone         == ""
  %s[0].fax           == ""
  %s[0].email         == ""
  %s[0].created_on    == nil
  %s[0].updated_on    == nil

#admin_contacts
  %s %CLASS{array}
  %s.size eq(1)
  %s[0] %CLASS{contact}
  %s[0].type          == Whois::Record::Contact::TYPE_ADMINISTRATIVE
  %s[0].id            == nil
  %s[0].name          == "Wild West Domains Wild West Domains"
  %s[0].organization  == "Wild West Domains"
  %s[0].address       == "14455 N Hayden Rd Suite 219"
  %s[0].city          == "Scottsdale"
  %s[0].zip           == "85260"
  %s[0].state         == "Arizona"
  %s[0].country       == nil
  %s[0].country_code  == "United States"
  %s[0].phone         == "+1.4805058800"
  %s[0].fax           == "+1.4805058844"
  %s[0].email         == "dns@wildwestdomains.com"
  %s[0].created_on    == nil
  %s[0].updated_on    == nil

#technical_contacts
  %s %CLASS{array}
  %s.size eq(1)
  %s[0] %CLASS{contact}
  %s[0].type          == Whois::Record::Contact::TYPE_TECHNICAL
  %s[0].id            == nil
  %s[0].name          == "Wild West Domains Wild West Domains"
  %s[0].organization  == "Wild West Domains"
  %s[0].address       == "14455 N Hayden Rd Suite 219"
  %s[0].city          == "Scottsdale"
  %s[0].zip           == "85260"
  %s[0].state         == "Arizona"
  %s[0].country       == nil
  %s[0].country_code  == "United States"
  %s[0].phone         == "+1.4805058800"
  %s[0].fax           == "+1.4805058844"
  %s[0].email         == "dns@wildwestdomains.com"
  %s[0].created_on    == nil
  %s[0].updated_on    == nil


#nameservers
  %s %CLASS{array}
  %s.size eq(3)
  %s[0] %CLASS{nameserver}
  %s[0].name == "cns1.secureserver.net"
  %s[1] %CLASS{nameserver}
  %s[1].name == "cns2.secureserver.net"
  %s[2] %CLASS{nameserver}
  %s[2].name == "cns3.secureserver.net"
